<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-PGJIAOP" failures="4" tests="12" name="EdgeTest" time="243.952" errors="0" timestamp="2024-01-21T10:19:14 IST">
  <testcase classname="automation.test.TestCase1" name="enteringCredentials" time="6.828"/>
  <testcase classname="automation.test.TestCase1" name="enteringCredentials" time="9.627"/>
  <testcase classname="automation.test.TestCase1" name="enteringCredentials" time="5.871"/>
  <testcase classname="automation.test.TestCase1" name="enteringCredentials" time="7.374"/>
  <testcase classname="automation.test.TestCase1" name="notEnteringCredentials" time="19.528">
    <skipped/>
  </testcase> <!-- notEnteringCredentials -->
  <testcase classname="automation.test.TestCase1" name="notEnteringCredentials" time="9.269">
    <failure type="java.lang.AssertionError" message="Element not found expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Element not found expected [true] but found [false]
at pageEvents.LoginPageEvents.verfiyIfLoginPageNotLoaded(LoginPageEvents.java:14)
at automation.test.TestCase1.notEnteringCredentials(TestCase1.java:40)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- notEnteringCredentials -->
  <testcase classname="automation.test.TestCase1" name="notEnteringCredentials" time="22.461">
    <skipped/>
  </testcase> <!-- notEnteringCredentials -->
  <testcase classname="automation.test.TestCase1" name="notEnteringCredentials" time="5.465">
    <failure type="java.lang.AssertionError" message="Element not found expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Element not found expected [true] but found [false]
at pageEvents.LoginPageEvents.verfiyIfLoginPageNotLoaded(LoginPageEvents.java:14)
at automation.test.TestCase1.notEnteringCredentials(TestCase1.java:40)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- notEnteringCredentials -->
  <testcase classname="automation.test.TestCase1" name="notEnteringCredentials" time="8.885">
    <skipped/>
  </testcase> <!-- notEnteringCredentials -->
  <testcase classname="automation.test.TestCase1" name="notEnteringCredentials" time="5.086">
    <failure type="java.lang.AssertionError" message="Element not found expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Element not found expected [true] but found [false]
at pageEvents.LoginPageEvents.verfiyIfLoginPageNotLoaded(LoginPageEvents.java:14)
at automation.test.TestCase1.notEnteringCredentials(TestCase1.java:40)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- notEnteringCredentials -->
  <testcase classname="automation.test.TestCase1" name="notEnteringCredentials" time="6.75">
    <skipped/>
  </testcase> <!-- notEnteringCredentials -->
  <testcase classname="automation.test.TestCase1" name="notEnteringCredentials" time="7.883">
    <failure type="java.lang.AssertionError" message="Element not found expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Element not found expected [true] but found [false]
at pageEvents.LoginPageEvents.verfiyIfLoginPageNotLoaded(LoginPageEvents.java:14)
at automation.test.TestCase1.notEnteringCredentials(TestCase1.java:40)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- notEnteringCredentials -->
</testsuite> <!-- EdgeTest -->
